test/WebJobs.Script.Tests/Binding/GeneralScriptBindingProviderTests.cs

```diff
@@ -21,27 +21,33 @@ public class GeneralScriptBindingProviderTests
         [InlineData(null, null, typeof(object))]
         [InlineData(null, "many", typeof(object[]))]
         [InlineData("string", null, typeof(string))]
-        [InlineData("StRing", null, typeof(string))] // case insenstive
-        [InlineData("string", "mANy", typeof(string[]))] // case insensitve
+        [InlineData("StRing", null, typeof(string))] // case insensitive
+        [InlineData("string", "mANy", typeof(string[]))] // case insensitive
         [InlineData("binary", null, typeof(byte[]))]
         [InlineData("binary", "many", typeof(byte[][]))]
         [InlineData("stream", null, typeof(Stream))]
         [InlineData("stream", "many", typeof(Stream[]))] // nonsense?
```

Comment:
nit - what does the question in the comment indicate? not sure how to interpret just first-pass reading over it

Reply:
I didn't write the original test so I'm not sure what the original author intended here? Potentially an example of a nonsense case?